kitsune/dashboards/readouts.py

```diff
@@ -500,7 +500,7 @@ def render(self, max_rows=None, rows=None):
             rows = self.rows(max_rows)
 
         # Compute percents for bar widths:
-        max_visits = max(r['visits'] for r in rows) if rows else 0
```

Comment:
Are we saying here that row['visits'] is None?

Reply:
Yeah, some rows are a dict like `{ ... visits: None, ... }`